src/Raven.Client/Documents/Linq/RavenQueryProviderProcessor.cs

```diff
@@ -2999,19 +3016,26 @@ private string TranslateSelectBodyToJs(Expression expression)
                     {
                         if (index > 0)
                             sb.Append(", ");
-                        
+
                         _jsProjectionNames.Add(name);
                         sb.Append(name).Append(" : ").Append(TranslateSelectBodyToJs(field.Expression));
                         continue;
                     }
-                   
+
                     AddJsProjection(name, field.Expression, sb, index != 0);
-                    
+
                 }
             }
 
+            if (expression is MemberExpression or MethodCallExpression)
+            { 
+                if (IsRawOrTimeSeriesCall(expression, out string script) == false)
+                    script = ToJs(expression);
+                
+                sb.Append(script);
+                return sb.ToString();
+            }
             sb.Append(" }");
```

Comment:
move `sb.Append(" }")` inside the `if` clauses

Reply:
Done